zboxcore/sdk/dirworker.go

```diff
@@ -116,7 +116,8 @@ func (req *DirRequest) commitRequest(existingDirCount int) error {
 			Timestamp:  timestamp,
 		}
 
-		commitReq.change = newChange
+		// commitReq.change = newChange
+		commitReq.changes = append(commitReq.changes, newChange)
```

Comment:
Should we not add operation for dirworker?

Reply:
Added